src/util/std_code.cpp

```diff
@@ -21,6 +23,10 @@ const irep_idt &code_deadt::get_identifier() const
   return to_symbol_expr(symbol()).get_identifier();
 }
 
+/// If this `codet` is not already a \ref code_blockt (i.e.\ it represents a
+/// single statement, not a block of statements), convert it to a
+/// \ref code_blockt with the original statement as its only operand.
```

Comment:
I'd add `; else return the unmodified input.`

Reply:
Reworded this so it now describes both cases.